lib/order.rb

```diff
@@ -0,0 +1,25 @@
+require './lib/takeaway.rb'
+class Order
+  attr_reader :items, :menu
+  def initialize(menu)
+    @items = {}
+    @menu = menu
+  end
+
+  def add_dish(dish, quantity = 1)
+    @items[dish] = 0 unless items[dish]
```

Comment:
You can define the hash on line 5 as @items = Hash.new(0), which means you don't need to put this line in